packages/hadron-build/commands/release.js

```diff
@@ -171,6 +171,27 @@ const writeLicenseFile = (CONFIG, done) => {
   });
 };
 
+// Remove a malicious link from chromium license
+// See: COMPASS-5333
+const fixCompass5333 = (CONFIG, done) => {
+  const appPackagePath = CONFIG.resourcesAppDir;
+
+  const chromiumLicensePath = path.join(
+    appPackagePath, 'node_modules',
+    'electron', 'dist', 'LICENSES.chromium.html'
+  );
+
+  const chromiumLicense = fs.readFileSync(chromiumLicensePath).toString();
```

Comment:
Optional, feel free to take it or leave it:
```suggestion
  const chromiumLicense = fs.readFileSync(chromiumLicensePath, 'utf8');
```